packages/mask/src/components/GuideStep/index.tsx

```diff
@@ -188,7 +188,7 @@ export default function GuideStep({
         <>
             {cloneElement(children as ReactElement<any>, { ref: childrenRef })}
             {usePortalShadowRoot((container) => {
-                if (!open || !clientRect.top || !clientRect.left) return null
+                if (!open || !clientRect?.top || !clientRect?.left) return null
```

Comment:
Why? Please reflect it on the type of `clientRect`.

Reply:
there's a crash.

Done.